slurm/install/rhel.sh

```diff
@@ -15,20 +15,19 @@ OS_VERSION=$(cat /etc/os-release  | grep VERSION_ID | cut -d= -f2 | cut -d\" -f2
 OS_ID=$(cat /etc/os-release  | grep ^ID= | cut -d= -f2 | cut -d\" -f2 | cut -d. -f1)
 
 
-if [ "$OS_VERSION" -gt "7" ]; then
-    if [ "${OS_ID,,}" == "rhel" ]; then
+if [ "$OS_VERSION" -gt "8" ]; then
+    PACKAGE_DIR=slurm-pkgs-rhel9
+else
+    PACKAGE_DIR=slurm-pkgs-rhel8
+fi
+
+if [ "${OS_ID,,}" == "rhel" ]; then
         dnf -y install -y perl-Switch
         dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
     else
         yum -y install epel-release
         dnf -y --enablerepo=powertools install -y perl-Switch
-    fi
-    PACKAGE_DIR=slurm-pkgs-rhel8
-else
-    echo "RHEL versions < 8 no longer supported"
-    exit 1
```

Comment:
Let's bring back this error message when trying to run rhel7

Reply:
Done